contributor-day.md

```diff
@@ -0,0 +1,102 @@
+
+### Accounts Setup
+1. [Required] Set up a WordPress.org account on the [account registration page](https://login.wordpress.org/register). If you get stuck, check out the [Learn WordPress lesson plan covering this topic](https://learn.wordpress.org/lesson-plan/creating-a-wordpress-org-account/) for guidance. And for users who already have a WordPress.org account, make sure you can still [log in](https://login.wordpress.org/) if it's been a while.
+2. [Recommended] Join the conversation on [WordPress Slack](https://make.wordpress.org/chat/) (must be logged on to WordPress.org). While Slack is not required for Contributor Day participation, it connects you directly with the talented and diverse community behind WordPress.
+3. [Optional] For users who intend to work with the [Gutenberg plugin](https://github.com/wordpress/gutenberg) or test PRs (pull requests), having a GitHub account is very helpful. You can [sign up for GitHub here](https://github.com/signup).
+
+### Local Test Environment
+Perhaps the biggest challenge faced by new contributors is preparing a test environment. If you would like to test patches and PRs directly during Contributor Day, it is **highly recommended** that you set things up at home *before* the event.
+
+> Tip: Wifi at the venue during a busy Contributor Day can be slow and unreliable, and configuring your laptop can take a lot (if not all) of your time. Don't wait to fork the [`wordpress-develop` repository](https://github.com/WordPress/wordpress-develop) or [download Docker](https://docs.docker.com/get-docker/) at the event, as this drain on bandwidth also impacts other contributors.
+
+Differences between operating systems and personal workflows make environment configuration beyond the scope of this article, but here are some links to get started on a local test/dev environment:
+
+- [Installing a Local Server](https://make.wordpress.org/core/handbook/tutorials/installing-a-local-server/): setting up Docker, Local, etc.
+- [Installing WordPress Locally](https://make.wordpress.org/core/handbook/tutorials/installing-wordpress-locally/): setting up WordPress using Git, SVN, or from a zip file.
+- [Get Set Up for Testing](https://make.wordpress.org/test/handbook/get-setup-for-testing/): environment setup for Gutenberg plugin PR testing.
+- [Apply a Patch from the Command Line](https://make.wordpress.org/core/handbook/tutorials/working-with-patches/#apply-a-patch-from-the-command-line) or [Download a GitHub PR](https://make.wordpress.org/core/handbook/tutorials/working-with-patches/#download-a-github-pull-request): common ways to test patches from Trac and GitHub.
+
+If you need help setting up your environment, reach out on [Make/Test Slack](https://wordpress.slack.com/archives/C03B0H5J0) and there will be other contributors who can provide additional guidance.
+
+**Local environment testing is suggested for past contributors, or users with prior development experience.**
+
+### Browser-Only Testing
```

Comment:
If I am not mistaken, if we use these options to get a test environment, testers will not be able to install patches/PRs. So in this case testers would be able to only participate in the reproducing the bugs, for example.

Reply:
Yes, the "Browser-Only Testing" section is to help users who may not have the technical knowledge (or time!) to set up a local environment.

I wonder, is it confusing to have the bolded text above the quoted code ("Local environment testing...") so close to the Browser-Only section? Should I move that bold text upward to be just under the "Local Test Environment" heading?

Reply:
I've moved the bolded guidelines directly under their corresponding headings to help avoid the possible confusion noted above.